modules/eks/cluster/README.md

```diff
@@ -150,6 +150,27 @@ components:
             addon_version: "v1.19.0-eksbuild.2"
 ```
 
+### Picking a Kubernetes Cluster Version
+
+When picking a Kubernetes version, you should consider the following:
+
+* When the end of life is. See [Amazon EKS End of Life Dates](https://endoflife.date/amazon-eks)
```

Comment:
```suggestion
### Amazon EKS End-of-Life Dates

When picking a Kubernetes version, be sure to review the [end-of-life dates for Amazon EKS](https://endoflife.date/amazon-eks). Refer to the chart below:
```

grammar and phrasing nitpick ... "consider the following - 1 thing". Why not just say the 1 thing directly?

Reply:
thought there was going to be more :)